Tw 256 Refresh /getting-started/navigating-postman/ (#3975)
postmanlabs/postman-docs: src/pages/docs/getting-started/navigating-postman.md

* **Invite**: If you have an [Admin role](/docs/collaborating-in-postman/roles-and-permissions/#workspace-roles) on a workspace, you can invite other users to collaborate.
* <img alt="Settings icon" src="https://assets.postman.com/postman-docs/icon-gear-outline-v9.jpg#icon" width="18px"> **Settings**: Access [Postman settings](/docs/getting-started/settings/) and other Postman resources.
* <img alt="Notifications icon" src="https://assets.postman.com/postman-docs/icon-notification-bell-v9.jpg#icon" width="18px"> **Notifications**: View recent activity from your team, get notifications about Postman updates, and see pull requests, comment activity, and other important information.
* **Your avatar**: View your profile, access your [account and notification settings](/docs/getting-started/postman-account/), see all active sessions for your account, or sign out of your account.
* **Team** (_paid plans_) or **Upgrade** (_free plan_): View resource usage and access your [billing dashboard](/docs/administration/billing/) and other account management tools.

### Search Postman

> To search in Postman, select __Search Postman__ in the header then enter your search terms. You can also use the keyboard shortcut `Command + K`. To change the scope of your search to your personal or team space or the Public API Network, select **All of Postman** and choose the desired scope. You can also change the scope of your search to **Workspaces**, **Collections**, **APIs**, or **Teams**.

Reviewer: ```suggestion
To search in Postman, select __Search Postman__ in the header then enter your search terms. You can also use the keyboard shortcut **⌘+K** or **Ctrl+K**. To change the scope of your search to your personal or team space or the Public API Network, select **All of Postman** and choose the desired scope. You can also change the scope of your search to **Workspaces**, **Collections**, **APIs**, or **Teams**.
```

Reviewer: Per Microsoft, use bold font for keyboard shortcuts (not code font) and use "⌘" instead of "Command" for Mac. Also make sure to include both the Mac and Windows keyboard shortcuts, as these are usually different.

> For signed-out users, search results only include public resources.

If you don't find what you are looking for in the list of results, select __Search all workspaces, collections, APIs and teams__ and you can find all the results in one page.

<img alt="Search all" src ="https://assets.postman.com/postman-docs/search-all-workspaces-collections-and-teams-v9.5.jpg"/>

> The __Search results__ page lists the scope and entity type. You can change these options to further filter your results.

Reviewer: You use both "entity" and "element" on this page. Have we standardized on one of these yet? If so, maybe just use that one. (My personal pref is "element" as "entity" sounds very programmer-y.)

Reviewer 2: We actually standardized on entity.

Author: Based on new conversation, I'm reverting all instances of "entity" on this page to "element" until we have a firmer decision.

You can also sort results using __Sort by__ on the right by selecting _Most relevant_ (default), _Most views_, or _Most recent_.

Depending on the element type, the search results contain different information:

* For _workspaces_, the search result contains the workspace type, summary, who published it, and when was it published.
* For _collections_, the search result contains the workspace type, whether the collection is a fork or not, who published it, and when was it published.
* For _APIs_, the search result contains the name and summary of the API, the API's owner (either an individual user or a team), and the workspace type.
* For _teams_, the search result contains the name and summary of the team. Selecting the team name redirects you to the team profile.

## Sidebar

> The Postman sidebar provides access to the fundamental elements of Postman:

Reviewer 3: ```suggestion
The left sidebar provides access to the fundamental elements of Postman:
```

* [Collections](/docs/getting-started/creating-the-first-collection/)
* [APIs](/docs/designing-and-developing-your-api/creating-an-api/)
* [Environments](/docs/sending-requests/managing-environments/#creating-environments)
* [Mock servers](/docs/designing-and-developing-your-api/mocking-data/setting-up-mock/)
* [Monitors](/docs/running-collections/scheduling-collection-runs/)
* [History](#history)

<img alt="Sidebar focused on Collections" src="https://assets.postman.com/postman-docs/navigating-postman-sidebar-v9.15.jpg" width="350px"/>

> To hide the sidebar, select the **hide sidebar icon** <img alt="Hide sidebar icon" src="https://assets.postman.com/postman-docs/icon-hide-sidebar-v9.jpg#icon" width="18px"> from the [footer](#footer).

Reviewer 3: I think the updated consensus is no bold for all icon directions

```suggestion
To hide the sidebar, select the hide sidebar icon <img alt="Hide sidebar icon" src="https://assets.postman.com/postman-docs/icon-hide-sidebar-v9.jpg#icon" width="18px"> from the [footer](#footer).
```

Author: Thank you - I missed this in my secondary sweep through!

### History

To access the requests you've made, select __History__ on the sidebar. When you're signed in to Postman, your history [syncs](/docs/getting-started/syncing/) across your devices.

> Select a request to open it again in a new tab. You can select more than one request by pressing `Command` or `Control` and selecting the requests.

Reviewer: ```suggestion
Select a request to open it again in a new tab. You can select more than one request by pressing **⌘** or **Ctrl** and selecting the requests.
```

<img alt="History Request" src="https://assets.postman.com/postman-docs/history-request-v9.jpg" width="350px"/>

> Your history also includes executed [collection runs](/docs/running-collections/intro-to-collection-runs/). These remain as the summarized version of the run and aren't logged as single requests.

Reviewer: ```suggestion
Your history also includes [collection runs](/docs/running-collections/intro-to-collection-runs/). These remain as the summarized version of the run and aren't logged as single requests.
```

* Select the **delete request icon** <img alt="Delete icon" src="https://assets.postman.com/postman-docs/icon-delete-v9.jpg#icon" width="12px"> to remove the request from your history.
* Select the **more actions icon** <img alt="Three dots icon" src="https://assets.postman.com/postman-docs/icon-three-dots-v9.jpg#icon" width="18px"> to access other actions, including creating a monitor, documentation, or mock server for the request.

#### Clearing your history

To remove all requests from your history, select the **more actions icon** <img alt="Three dots icon" src="https://assets.postman.com/postman-docs/icon-three-dots-v9.jpg#icon" width="18px"> next to the History search bar, then select **Clear all**.

#### Saving responses in history

To save request responses in your history, select the **more actions icon** <img alt="Three dots icon" src="https://assets.postman.com/postman-docs/icon-three-dots-v9.jpg#icon" width="18px"> next to the History search bar, then turn on **Save Responses**.

> You can't use the __Save Responses__ option with requests from [Collection Runner](/docs/running-collections/intro-to-collection-runs/).

## Main work area

> Whether you're working with a collection, an API, or another entity type, the center section of Postman is where you do the majority of your work. [Tabs](#tabs) enable you to organize your work, while the right sidebar, the [right sidebar](#right-sidebar) gives you access to entity-specific tools like documentation, and the [environment selector and environment quick look](#environment-selector-and-environment-quick-look) enable you to manage variables.

Reviewer: ```suggestion
Whether you're working with a collection, an API, or another entity type, the center section of Postman is where you do the majority of your work. [Tabs](#tabs) enable you to organize your work, while the [right sidebar](#right-sidebar) gives you access to entity-specific tools like documentation, and the [environment selector and environment quick look](#environment-selector-and-environment-quick-look) enable you to manage variables.
```

### Tabs

Tabs allow you to organize and work between requests.

#### Opening a new tab

To open a new tab, select __+__ in the middle of the screen.

> > **Keyboard shortcut**: You can also select **Command + T** or **Ctrl + T** on your keyboard to open a new tab.

Reviewer: ```suggestion
> You can also select **⌘+T** or **Ctrl+T** to open a new tab.
```

#### Saving or discarding changes

If a tab has unsaved changes, Postman displays a dot next to the tab name. Select <img alt="Save icon" src="https://assets.postman.com/postman-docs/icon-save.jpg#icon" width="16px"> **Save** to save the changes. To close the tab and discard changes, select the **close icon** <img alt="Close icon" src="https://assets.postman.com/postman-docs/icon-close.jpg#icon" width="16px"> then select **Don't save**.

#### Viewing conflicts

A tab will alert you to a conflict if you or a collaborator changes its contents in another tab or workspace. Postman prompts you to resolve any conflicts that happen.

<img alt="Tab displaying a conflict" src="https://assets.postman.com/postman-docs/tab-conflict.jpg" width="300px"/>

#### Managing tabs

You can have many tabs open at the same time. To rearrange your open tabs, select and drag them in the desired order.

> To manage your open tabs or to access recently closed tabs, select the three dots <img alt="Three dots icon" src="https://assets.postman.com/postman-docs/icon-three-dots-v9.jpg#icon" width="18px"> to open a menu with action options:

Reviewer: Use "more option" instead of "three dots" and make it bold (to match elsewhere on the page)?

> * **Recently Closed Tabs**: Shows you tabs that you closed recently. Select one to reopen it.

Reviewer: ```suggestion
* **Recently Closed Tabs**: Shows tabs that you closed recently. Select one to reopen it.
```

> * **Duplicate Selected Tab**: _(Desktop app only.)_ Duplicates the current tab. This doesn't create a copy of the request, so when you duplicate a tab any edits you make are reflected in the original request.

Reviewer: ```suggestion
* **Duplicate Selected Tab**: _(Desktop app only)_ Duplicates the current tab. This doesn't create a copy of the request, so when you duplicate a tab any edits you make are reflected in the original request.
```

> * **Close Selected Tab**: Closes the current tab. If the tab has unsaved changes, Postman will ask if you want to save before closing.

Reviewer: ```suggestion
* **Close Selected Tab**: Closes the current tab. If the tab has unsaved changes, Postman asks if you want to save before closing.
```

> You can also manage your open tabs or open a new tab by right-clicking on an existing tab:

Reviewer: I fell like you could turn this statement into a note and remove the bullet list that follows, as it's largely repetition of the list above.

Author: Yes, agreed! I'll simplify this section since it's mostly duplicated information.

* **New Request**: _(Postman for Web only.)_ Opens a new tab.
* **Duplicate Tab**: _(Desktop app only.)_ Duplicates the current tab. This doesn't create a copy of the request, so when you duplicate a tab any edits you make are reflected in the original request.
* **Close Tab**: Closes the current tab. If the tab has unsaved changes, Postman will ask if you want to save before closing.
* **Force Close Tab**: Closes the current tab without saving any changes.
* **Close All Tabs**: Closes all tabs. If any tabs contain unsaved changes, Postman will ask if you want to save before closing.
* **Force Close All Tabs**: Closes all tabs without saving any changes.

<img alt="Manage tabs" src="https://assets.postman.com/postman-docs/right-click-manage-tab.jpg" width="300px"/>

> > **Closing unsaved tabs**: You can set whether Postman asks whether you want to save when you close a tab that has changes. Select the gear icon <img alt="Settings icon" src="https://assets.postman.com/postman-docs/icon-gear-outline-v9.jpg#icon" width="18px"> to open **Settings**. Under **Requests**, use the toggle next to **Always ask when closing unsaved tabs** to turn this option on or off.

Reviewer: Use "settings icon" instead of "gear icon" and make it bold, as elsewhere on the page?

If you have a lot of tabs open, they might overflow the area of the tab bar. To navigate to tabs that are outside the viewable area, select the arrows to the left or right of the tab bar.

<img alt="Scroll to navigate tabs" src="https://assets.postman.com/postman-docs/tab-navigation-scroll.jpg" width="800px"/>

> ### Right sidebar

Reviewer: Is this the preferred term now? On other pages it's referred to as the context bar.

Reviewer 2: We should call it right sidebar for now, but that's still being discussed. (Don't change any other pages yet on this.)

**Pull requests** <img alt="Pull request icon" src="https://assets.postman.com/postman-docs/icon-pull-request.jpg#icon" width="16px"> | Collections, environments | View any [pull requests](/docs/collaborating-in-postman/version-control/#creating-pull-requests) for a Postman entity.
**Forks** <img alt="Fork icon" src="https://assets.postman.com/postman-docs/icon-fork.jpg#icon" width="14px"> | Collections, environments | View any [forks](/docs/collaborating-in-postman/version-control/#forking-postman-entities) of a Postman entity.
**Activity feed** <img alt="Activity feed icon" src="https://assets.postman.com/postman-docs/icon-activity-feed-v9.jpg#icon" width="16px"> | Monitors | View the activity feed for a [monitor](/docs/monitoring-your-api/intro-monitors/).

### Environment selector and environment quick look

> The environment selector enables you to choose which [environment](docs/sending-requests/managing-environments/) to use in your work. Select an environment from the menu to set it as the active environment, which gives you access to the variables in that environment.

Reviewer: ```suggestion
The environment selector enables you to choose which [environment](/docs/sending-requests/managing-environments/) to use in your work. Select an environment from the menu to set it as the active environment, which gives you access to the variables in that environment.
```

Author: Eagle eyes catch! 👓 🦅

<img src="https://assets.postman.com/postman-docs/environment-selector-v9.13.jpg" alt="Environment selector" width="300">

To check a variable value at a glance, use the **quick look icon** <img alt="Quick Look icon" src="https://assets.postman.com/postman-docs/eye.jpg#icon" width="16px">.

## Footer

The footer on the bottom of Postman enables you to find and replace text, open the console, capture requests and cookies, open __Bootcamp__, and access several other tools.

<img alt="Postman footer left view" src="https://assets.postman.com/postman-docs/navigating-postman-footer-left-v9.jpg" width="300px"/>

* <img alt="Hide sidebar icon" src="https://assets.postman.com/postman-docs/icon-hide-sidebar-v9.jpg#icon" width="18px"> **Hide sidebar**: Close or reopen the [sidebar](#sidebar).

> * <img alt="Search icon" src="https://assets.postman.com/postman-docs/icon-search-v9.jpg#icon" width="16px"> **Find and replace**: _Postman Desktop app only._ Search the current workspace. You can also use the shortcuts __Command + Shift + F__ or __Ctrl + Shift + F__. Enter your search string then select **Find**. Limit your search to a specific entity type by selecting **Collections**, **Environments**, **Globals**, or **Open tabs**. To replace your search term in a selected entity, select **Replace in selected**.

Reviewer: ```suggestion
* <img alt="Search icon" src="https://assets.postman.com/postman-docs/icon-search-v9.jpg#icon" width="16px"> **Find and replace**: (_Postman Desktop app only_) Search the current workspace. You can also use the shortcuts **⌘+Shift+F** or **Ctrl+Shift+F**. Enter your search string then select **Find**. Limit your search to a specific entity type by selecting **Collections**, **Environments**, **Globals**, or **Open tabs**. To replace your search term in a selected entity, select **Replace in selected**.
```
